src/resource_manager/http_route.py

```diff
@@ -139,6 +144,21 @@ def _gen_resource(self, resource_definition: ResourceDefinition) -> GenericNames
                             }
                         }
                     ],
+                    "filters": (
+                        []
+                        if not http_route_resource_definition.strip_prefix
```

Comment:
Wait, this is inversed no ? It should be `if http_route_resource_definition.strip_prefix`

Reply:
nevermind